.gitignore

```diff
@@ -74,4 +74,6 @@ keymanager.db
 # Contract ethers typechain
 ethers-types
 
-config
\ No newline at end of file
+common/config/

```

Comment:
This is the rush config right, wouldn't we want to commit this to vc?

Reply:
Good question, maybe we do want it committed 🤔

Reply:
Removed this line so that the config files are added 👍